package.json

```diff
@@ -90,7 +90,7 @@
   "homepage": "https://github.com/warp-contracts/warp#readme",
   "dependencies": {
     "archiver": "^5.3.0",
-    "arweave": "1.13.7",
+    "arweave": "1.14.4",
```

Comment:
`getByHeight` API for blocks is exposed in 1.14.0 - may need to validate there are no other breaking changes